libs/ng-mocks/src/lib/common/core.form.ts

```diff
@@ -1,14 +1,14 @@
-import * as module from '@angular/forms';
+import * as m from '@angular/forms';
 
-const AbstractControl = module.AbstractControl;
-const DefaultValueAccessor = module.DefaultValueAccessor;
-const FormControl = module.FormControl;
-const FormControlDirective = module.FormControlDirective;
-const NG_ASYNC_VALIDATORS = module.NG_ASYNC_VALIDATORS;
-const NG_VALIDATORS = module.NG_VALIDATORS;
-const NG_VALUE_ACCESSOR = module.NG_VALUE_ACCESSOR;
-const NgControl = module.NgControl;
-const NgModel = module.NgModel;
```

Comment:
please, don't rename things. Only add changes which are needed for the fix itself and to test it.